convert2rhel/unit_tests/logger_test.py

```diff
@@ -167,7 +167,7 @@ def test_logfile_buffer_handler(read_std):
 
     stdouterr_out, _ = read_std()
     assert "message 1" not in stdouterr_out
-    assert "message 2" in stdouterr_out
+    assert "message 2" not in stdouterr_out
```

Comment:
@Venefilyn, I was getting the following unit test failure. Is this fix all right?
```
=================================== FAILURES ===================================
_________________________ test_logfile_buffer_handler __________________________
[gw0] linux2 -- Python 2.7.5 /usr/bin/python2

read_std = <function factory at 0x7f27c05fd7d0>

    @pytest.mark.noautofixtures
    def test_logfile_buffer_handler(read_std):
        logbuffer_handler = logger_module.LogfileBufferHandler(2, "custom_name")
        logger = logging.getLogger("convert2rhel")
        logger.addHandler(logbuffer_handler)
    
        logger.warning("message 1")
        logger.warning("message 2")
    
        # flushing without other handlers should work, it will just go to NullHandlers
        logbuffer_handler.flush()
    
        stdout_handler = logging.StreamHandler(sys.stdout)
        stdout_handler.name = "custom_name"
        logger.addHandler(stdout_handler)
    
        # flush to the streamhandler we just created
        logbuffer_handler.flush()
    
        stdouterr_out, _ = read_std()
        assert "message 1" not in stdouterr_out
>       assert "message 2" in stdouterr_out
E       AssertionError: assert 'message 2' in ''

convert2rhel/unit_tests/logger_test.py:170: AssertionError
```

Reply:
Interestingly, the unit tests executed here on GitHub are giving me an opposite result:
```
=================================== FAILURES ===================================
_________________________ test_logfile_buffer_handler __________________________
[gw2] linux2 -- Python 2.7.5 /usr/bin/python2

read_std = <function factory at 0x7ff1b0f57c80>

    @pytest.mark.noautofixtures
    def test_logfile_buffer_handler(read_std):
        logbuffer_handler = logger_module.LogfileBufferHandler(2, "custom_name")
        logger = logging.getLogger("convert2rhel")
        logger.addHandler(logbuffer_handler)
    
        logger.warning("message 1")
        logger.warning("message 2")
    
        # flushing without other handlers should work, it will just go to NullHandlers
        logbuffer_handler.flush()
    
        stdout_handler = logging.StreamHandler(sys.stdout)
        stdout_handler.name = "custom_name"
        logger.addHandler(stdout_handler)
    
        # flush to the streamhandler we just created
        logbuffer_handler.flush()
    
        stdouterr_out, _ = read_std()
        assert "message 1" not in stdouterr_out
>       assert "message 2" not in stdouterr_out
E       AssertionError: assert 'message 2' not in 'message 2\n'
E         'message 2' is contained here:
E         Strings contain only whitespace, escaping them using repr()
E           'message 2\n'

convert2rhel/unit_tests/logger_test.py:170: AssertionError
```

Reply:
I just pulled down the PR locally and ran without any issues :thinking: 
```
============= 1323 passed, 97 skipped, 1 warnings in 8.32 seconds ==============
```